guests/__init__.py

```diff
@@ -20,3 +20,27 @@
 c.MENU['People'].append_menu_item(
     MenuItem(access=c.BANDS, name='Guests', href='../guest_admin/?filter=only-guests')
 )
+
+
+@JinjaEnv.jinja_filter
+def comma_join(xs, conjunction='and'):
```

Comment:
This is super useful! So useful, in fact, I think it should probably go in the main plugin?

Reply:
Yep! Laziness on my part!